app/utils/initialize/websocket-server.js

```diff
@@ -0,0 +1,29 @@
+const javaScriptUtils = require("../javascript-utils");
+
+module.exports = function(app, webSocketUserSettings) {
+  app.ws("/", ws => {
```

Comment:
This is one of the app's main business logic, should definitely not be in the utils folder.